srcjs/jster.ts

```diff
@@ -148,14 +155,26 @@ class Jster {
     );
   }
 
+  waitFor(ms) {
+    this.add((done) => {
+      setTimeout(done, ms);
+    });
+  }
+  waitForShiny() {
```

Comment:
Might want to replace this with event handlers that listen for `shiny:idle` and `shiny:busy` events. https://shiny.rstudio.com/articles/js-events.html